src/effects/builtin/whitenoiseeffect.h

```diff
@@ -0,0 +1,43 @@
+#ifndef WHITENOISEEFFECT_H
+#define WHITENOISEEFFECT_H
```

Comment:
again: \#pragma once

Reply:
What is the solution? I saw other effects do the same

Reply:
https://en.wikipedia.org/wiki/Include_guard
#pragma once is easier and less error-prone but wasn't universally supported until recently, so mixxx codebase is still full of the "old" format. 
New code should use #pragma once

Reply:
Thank you. I will implement that. The build is failing, but from the errors I see this is not from this PR, is that correct?

Reply:
Just replace these two lines with "#pragma once" and remove the "#endif" at the bottom.

> The build is failing, but from the errors I see this is not from this PR, is that correct?

If the build worked before and you didn't pull in changes from other branches, they have to be from this PR.

Reply:
Hm, I am not sure the build has worked before. Some of the automated tests succeed though. And it always compiles on my pc :)

Reply:
I didn't realize you were talking about the CI builds. It still builds on bionic which doesn't have QRandomGenerator because the qt version is too old.

Reply:
Yeah, this should be fine now I hope